speech/transcript/transcript.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2015 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Command transcript reads an audio file and outputs the transcript for it.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"golang.org/x/net/context"
+	"google.golang.org/api/option"
+	"google.golang.org/api/transport"
+	speech "google.golang.org/genproto/googleapis/cloud/speech/v1"
+)
+
+const usage = `Usage: transcript <audiofile>
+
+Audio file is required to be 16-bit signed little-endian encoded
+with a sample rate of 16000.
+`
+
+func main() {
+	flag.Parse()
+	if len(os.Args) < 2 {
+		fmt.Fprintln(os.Stderr, usage)
+		os.Exit(2)
+	}
+
+	ctx := context.Background()
+	conn, err := transport.DialGRPC(ctx,
+		option.WithEndpoint("speech.googleapis.com:443"),
+		option.WithScopes("https://www.googleapis.com/auth/cloud-platform"),
+	)
```

Comment:
Actually, it seems pretty fair to use it here until we have the generated gRPC clients. The huge plus is handling ADC.